includes/wc-payment-api/class-wc-payments-api-client.php

```diff
@@ -1009,6 +1009,31 @@ public function get_login_data( $redirect_url ) {
 		);
 	}
 
+	/**
+	 * Get a one-time capital link.
+	 *
+	 * @param string $type        The type of link to be requested.
+	 * @param string $return_url  URL to navigate back to from the dashboard.
+	 * @param string $refresh_url URL to navigate to if the link expired, has been previously-visited, or is otherwise invalid.
+	 *
+	 * @return array Account link object with create, expires_at, and url fields.
+	 *
+	 * @throws API_Exception When something goes wrong with the request, or there aren't valid loan offers for the merchant.
+	 */
+	public function get_capital_link( $type, $return_url, $refresh_url ) {
```

Comment:
I think this was already created by Marcin here: 1382-gh-Automattic/woocommerce-payments-server

Reply:
I just see the server endpoint added in that PR. I also searched the client for any mention of a request to `capital_links`, but couldn't find anything except this method, which is what makes the request to the server method you mentioned above. Am I missing something here?

Reply:
This creates an internal REST request to the endpoint created by Marcin, and that endpoint returns the link to this method as the method output. I believe the server endpoint was created to create a link to fulfill a request from the client, before the designs showed us the e-mail flow, I think it'll be good to eliminate the internal REST request and directly call the Stripe API, how does it sound? /cc @marcinbot